docs/user/dashboard/dashboard.asciidoc

```diff
@@ -4,518 +4,669 @@
+
+.. In the *Display name* field, enter `Transferred bytes`.
 
-View data in a different <<index-patterns,index pattern>>, and change the data label name and colors. The options available depend on the panel type.
+.. From the *Value format* dropdown, select `Bytes (1024)`.
 
-To change the index pattern, click *Panel options*, then enter the new *Index Pattern*.
+.. Click *Close*.
 
-To override the index pattern for a data set, click *Data > Options*. Select *Yes* to override, then enter the new *Index pattern*.
+. From the *Available fields* list, drag and drop *hour_of_day* to *Horizontal axis* in the editor, then configure the options.
 
-To change the data labels and colors:
+.. Click *hour_of_day*.
 
-. Click *Data*.
-. Enter the *Label* name, which *TSVB* uses on the legends and data labels.
-. Click the color picker, then select the color for the data.
+.. Click and slide the *Intervals granularity* slider until the horizontal axis displays hourly intervals.
 +
 [role="screenshot"]
-image::images/tsvb_color_picker.png[TSVB color picker]
+image::images/lens_end_to_end_5_2.png[Create custom ranges]
 
 [float]
-[[add-annotations]]
-==== Add annotations
+==== Save the panel
+
+Save and add the visualization panel to the dashboard.
+
+. From the toolbar, click *Save*.
+
+. In the *Title* field, enter `Hourly Traffic Distribution`. 
+
+. From the *Tags* drop down, select any applicable tags. 
+
+. Select *Add to Dashboard after saving*.
+
+. Click *Save and return*.
+
+[discrete]
+[[custom-ranges]]
+=== View the percent of small versus large transferred files 
+ 
+To determine if your users transfer more small files versus large files, create a pie chart that displays the percentage of each size. 
+
+. Open *Lens*.
+
+. From the *Available fields* list, drag and drop *bytes* to *Vertical axis* in the editor, then configure the options.
+
+.. Click *Average of bytes*.
+
+.. From *Select a function*, click *Sum*.
+
+.. Click *Close*.
+
+. From the *Available fields* list, drag and drop *bytes* to *Break down by* in the editor, then specify the file size ranges.
+
+.. Click *bytes*.
+
+.. Click *Create custom ranges*, enter the following, then press Return:
+
+* *Ranges* &mdash; `0` -> `10240`
+
+* *Label* &mdash; `Below 10KB`
+
+.. Click *Add range*, enter the following, then press Return:
+
+* *Ranges* &mdash; `10240` -> `+∞`
+
+* *Label* &mdash; `Above 10KB`
++
+[role="screenshot"]
+image::images/lens_end_to_end_6_1.png[Custom ranges configuration]
 
-You can overlay annotation events on top of your *Time Series* charts. The options available depend on the data source.
+.. From the *Value format* dropdown, select *Bytes (1024)*.
 
-To begin, click *Annotations*, click *Add data source*, then configure the options.
+.. Click *Close*.
 
+. From the *Chart Type* dropdown, select *Pie*.
++
 [role="screenshot"]
-image::images/tsvb_annotations.png[TSVB annotations]
+image::images/lens_end_to_end_6_2.png[Files size distribution]
```

Comment:
This screenshot looks like it's still using the compatibility palette, can we change it to the default one?